### DISCUSSION TITLE
Why there's no new Who's hiring posts?

### DISCUSSION THREAD

**Discussion by Stupidism** [vercel/next.js#58365] [Help] [answered]
### Summary

I saw there was a post every month.

**Comment by InsaneNaman** [comment]
I guess a couple of folks spammed too much. That's why no more hiring posts. HN's post is live tho -https://news.ycombinator.com/item?id=38490811

**Comment by leerob** [comment, marked as answer]
We're pausing doing these posts for now - we might restart them in the future 👍
